devops/scripts/deploy-decoupled-upstream.sh

```diff
@@ -40,10 +40,13 @@ for commit in $newcommits; do
   # Exclude commits which have been manually rejected
   skip=false
   for item in "${exclude_list[@]}"; do
-    [[ $item == $commit ]] && echo "Commit ${commit} has been manually excluded."; skip=true
+    if [[ $item == $commit ]]; then
+      echo "Commit ${commit} has been manually excluded."
+      skip=true
+    fi
   done
 
-  if [[ $skip==true ]] ; then
+  if [[ $skip = true ]] ; then
```

Comment:
@pwtyler is there a distinction between `$variable==true` and `$variable == true`? In our testing, _not_ making this change resulted in all subsequent commit hashes being marked `true` to `$skip`.

Reply:
Tested locally and confirmed this was working as expected. So, I guess that answers my question about `$variable==true` vs `$variable == true` cc @rwagner00

Reply:
bash `test` with brackets can be very picky about whitespace— it may have been doing something also off without the spaces as well.

There are other safety steps we could take (i.e. I generally wrap all variables in ""), but ^ should be fairly safe as we're explicitly setting/resetting $skip in the loop.